feat: Added interactive globe control options and enhanced playground

RamanaRaj7 opened the pull request:
This commit introduces new interactive features to the COBE globe:

- Added draggable functionality with X and Y axis control
- Implemented auto-rotation with configurable speed
- Added return-to-default position or axis behavior after drag is released
- Added showcases demonstrating new features:
  - Auto-rotate globe example
  - Draggable globe (horizontal/vertical/omnidirectional)
  - Return-to-position functionality
- Updated API documentation with new parameters and methods
- Added new globe instance methods: toggleRotation(), isRotating(), resetPosition()

fixes: fixed the marker size in the playground (size was not changing) and added better font handling

shuding commented:
Really appreciate your contributions but this lib is intended to be a small and lightweight core, while other functionalities can be built on top of it (see the examples on the docs).

It might be great to have a more opinionated globe component on top of COBE with helpful utilities, either as a fork or a new lib.

Thanks again!

RamanaRaj7 commented:
> Thanks for your comments! really appreciate it. will look into creating the new lib with more options which extends over this. But what about this fix: fixed the marker size in the playground (size was not changing) and added better font handling?

should I create a separate PR for that?
